src/text-fragment-utils.js

```diff
@@ -649,7 +685,8 @@ const normalizeString = (str) => {
   return (str || '')
     .normalize('NFKD')
     .replace(/\s+/g, ' ')
-    .replace(/[\u0300-\u036f]/g, '');
+    .replace(/[\u0300-\u036f]/g, '')
+    .toLowerCase();
```

Comment:
Do we have access to any locale value(s)? I'm worried about the turkish i.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLocaleLowerCase

Reply:
I don't think we do--that stuff is generally provided to the server in the request headers, and not to the client in the document. I don't think using the user's browser locale is appropriate, as that creates different behavior for different users accessing the same URL.

I *think* we avoid creating any false negatives here because `İ` and `I` both resolve to `i` once the diacritics are removed (I added a test for this). We may introduce a few false positives where a page contains two words that are identical except one has a capital `I` and one has a capital `İ`.

Reply:
Gotcha, thanks for adding the test.